tests/components/energy/test_sensor.py

```diff
@@ -41,6 +42,13 @@ async def setup_integration(hass):
     return setup_integration
 
 
+@pytest.fixture(autouse=True)
+@freeze_time("2022-04-19 07:53:05")
+def frozen_time():
+    """Freeze clock for tests."""
+    yield
+
+
```

Comment:
A bit unclear if this is needed after the garbage collect change. But i still think it make sense.